spark/src/main/scala/io/substrait/spark/expression/ToSubstraitExpression.scala

```diff
@@ -141,10 +143,6 @@ abstract class ToSubstraitExpression extends HasOutputStack[Seq[Attribute]] {
       case SubstraitLiteral(substraitLiteral) => Some(substraitLiteral)
       case a: AttributeReference if currentOutput.nonEmpty => translateAttribute(a)
       case a: Alias => translateUp(a.child)
-      case p
```

Comment:
why this removal? I'd be fine with stopping support for 3.2 myself, but that should be a separate PR if possible

Reply:
I'd removed this at some point to get rid of an annoying NullPointerException, then forgot to follow up with why.  I've added it back in with a NULL safe string equality check.